main.go

```diff
@@ -40,17 +43,29 @@ func main() {
 	var enableLeaderElection bool
 	var namespace string
 	var argocdRepoServer string
+	var duration int64
+	var debugLog bool
+	var updateSync bool
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&metricsAddr, "probe-addr", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.StringVar(&namespace, "namespace", "argocd", "Argo CD repo namesapce")
 	flag.StringVar(&argocdRepoServer, "argocd-repo-server", "argocd-repo-server:8081", "Argo CD repo server address")
+	flag.Int64Var(&duration, "git-refresh-duration", 60, "(seconds) The refrash duration for the git generator")
+	flag.BoolVar(&debugLog, "debug", false, "print debug logs")
+	flag.BoolVar(&updateSync, "sync-update", true, "if false, then the controller will only create and delete application, and will not update existing applications.")
```

Comment:
Can you clarify how this would be used? I'm not sure why anyone would want to generate apps but then not have them reconciled with changes in git.

Reply:
Hi, I created a seprate PR for this and explaind the use case there: https://github.com/argoproj-labs/applicationset/pull/48

Reply:
Good idea. Thanks.